src/components/toolbar/toolbar.spec.ts

```diff
@@ -20,45 +15,32 @@ describe('MdToolbar', () => {
     TestBed.compileComponents();
   }));
 
-  beforeEach(inject([TestComponentBuilder], (tcb: TestComponentBuilder) => {
-    builder = tcb;
-  }));
-
-  it('should apply class based on color attribute', (done: () => void) => {
-    return builder.createAsync(TestApp).then((fixture) => {
-      let testComponent = fixture.debugElement.componentInstance;
-      let toolbarDebugElement = fixture.debugElement.query(By.css('md-toolbar'));
-
-      testComponent.toolbarColor = 'primary';
-      fixture.detectChanges();
+  it('should apply class based on color attribute', () => {
+    let fixture = TestBed.createComponent(TestApp);
+    let testComponent = fixture.debugElement.componentInstance;
+    let toolbarDebugElement = fixture.debugElement.query(By.css('md-toolbar'));
 
-      expect(toolbarDebugElement.nativeElement.classList.contains('md-primary')).toBe(true);
+    testComponent.toolbarColor = 'primary';
+    fixture.detectChanges();
 
-      testComponent.toolbarColor = 'accent';
-      fixture.detectChanges();
+    expect(toolbarDebugElement.nativeElement.classList.contains('md-primary')).toBe(true);
 
-      expect(toolbarDebugElement.nativeElement.classList.contains('md-primary')).toBe(false);
-      expect(toolbarDebugElement.nativeElement.classList.contains('md-accent')).toBe(true);
+    testComponent.toolbarColor = 'accent';
+    fixture.detectChanges();
 
-      testComponent.toolbarColor = 'warn';
-      fixture.detectChanges();
+    expect(toolbarDebugElement.nativeElement.classList.contains('md-primary')).toBe(false);
+    expect(toolbarDebugElement.nativeElement.classList.contains('md-accent')).toBe(true);
 
-      expect(toolbarDebugElement.nativeElement.classList.contains('md-accent')).toBe(false);
-      expect(toolbarDebugElement.nativeElement.classList.contains('md-warn')).toBe(true);
+    testComponent.toolbarColor = 'warn';
+    fixture.detectChanges();
 
-      done();
-    });
+    expect(toolbarDebugElement.nativeElement.classList.contains('md-accent')).toBe(false);
+    expect(toolbarDebugElement.nativeElement.classList.contains('md-warn')).toBe(true);
   });
-
 });
 
-@Component({
-  selector: 'test-app',
-  template: `
-    <md-toolbar [color]="toolbarColor">
-      <span>Test Toolbar</span>
-    </md-toolbar>`
-})
+
+@Component({template: `<md-toolbar [color]="toolbarColor"><span>Test Toolbar</span></md-toolbar>`})
```

Comment:
Not necessary, but since the template is now in a single line, we could probably remove the `span` element to make the template more clean.